src/java/picard/analysis/CollectQualityYieldMetrics.java

```diff
@@ -45,14 +45,41 @@
  *
  * @author Martha Borkan
  */
+
+
 @CommandLineProgramProperties(
-        usage = "Collects quality yield metrics, a set of metrics that quantify the quality and yield of sequence data from a " +
-                "SAM/BAM input file.",
-        usageShort = "Collects a set of metrics that quantify the quality and yield of sequence data from the provided SAM/BAM",
+        usage = CollectQualityYieldMetrics.USAGE_SUMMARY + CollectQualityYieldMetrics.USAGE_DETAILS,
+        usageShort = CollectQualityYieldMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectQualityYieldMetrics extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Collects a set of metrics that indicate the quantity of reads that pass quality thresholds and Illumina-specific filters.  ";
+    static final String USAGE_DETAILS = "This tool helps the user determine the overall quality of a set of reads " +
+            "determining how many reads of a run have a minimum base-quality score and pass filters.  Either the original (OQ) or the recalibrated base quality " +
+            "scores can be used.  The recalibrated base quality scores are obtained from the Base Quality Score Recalibration " +
```

Comment:
First sentence is missing something grammatically.  Also, it says this is calculated on reads of a "run," but this tool is usually used on a read group bam that represents a lane or a fraction or a lane of sequence data, not necessarily an entire run.